### DISCUSSION TITLE
Lesson 11: file not uploaded to IPFS

### DISCUSSION THREAD

**Discussion by Karlus44** [smartcontractkit/full-blockchain-solidity-course-py#1419] [Q&A] [answered]
Hello,
I'm trying to upload my image to IPFS, I got this address:

You have created 1 collectibles !
Creating metadata file ./metadata/rinkeby/0-ST_BERNARD.json
https://ipfs.io/ipfs/QmWkHWDCRet4gwRqEmRTokP4gPdV4dkjeTKRPK4qAufFGt?filename=st-bernard.png

Yet, the image file doesn't seem to have been uploaded correctly. When I load this page I got an error.
My code is here if you need to check:
https://github.com/Karlus44/nft-demo
Thanks for any help !

**Comment by Karlus44** [comment, marked as answer]
ok, nevermind. The problem comes from my img file which was incorrect

**Reply by cromewar** [reply]
I'm glad you figured out so quickly, feel free to make any other questions.
